election-guidelines-draft.md

```diff
@@ -0,0 +1,73 @@
+
+For the first year of its ONNX open governance, representitives from Facebook, Microsoft, AWS, Intel and Nvidia are chosen to serve as the ONNX Steering Committee to help guide the project. The Steering Committee will be elected by the [Contributors](https://github.com/onnx/onnx/tree/master/community#community-roles) in its second year and will be re-elected every year. 
+
+This document is created to provide guidelines for the election process to ensure maximum transparency and fairness. 
+
+
+## Timeline
+
+Candidate applications will be accepted in April, and the election will be held in May. The new term for Steering Committee begins on June 1st of the corresponding year. The following table outlines the schedule for the election process. 
+
+| Schedule     | Event               | 
+|:-------------|:--------------------|
+| 1st Monday of April| Application for Steering Committee candidates open. |  
+| 3rd Monday of April| Candidates and their applications posted on github. Campaigning session held over VC.| 
+| 1st Monday of May| Election begins.     |
+| 2nd Monday of May| Election closes, and votes counted. Election results announced in the same week.|    
+| 3rd Monday of May| Previous Steering Committee to meet the newly elected Committee for official transition.| 
+| June 1 | New term begins with elected Steering Committee. Steering Committee Emeritus members help with the transition for the month of June.      |
+
+
+## Eligibility 
+
+### Eligibility for Steering Committee candadicy
+
+<code>**[TODO]** Candadicy eligbility needs to be discussed </code>
+
+Candidates can either be self-nominated or endorsed by one of Member Companies. A self-nominated candidate must at least be a contributor. 
+
+ 
+### Eligibility for voting
+
+To participate in the Steering commitee election, you must be a Contributor to the ONNX project. As defined in the community guideline, Contributor is sponsored by 2 approvers from different companies. 
+
+Contributors are further required to submit their github handle, email address, and affiliated company name to be eligible for voting. Any Contributor who has not submitted their information by before April 31st will not be able to participate in the election. 
+
+<code>**[TODO]** There needs to be a list of contributors</code> 
+
+## Candidacy process
+
+## Voting process 
+
+### General election procedure
+In order to promote fairness, the Steering Committee has decided to limit 1 vote per Member Company. Contributors will be able to vote individually, but their votes will be rolled up to represent the vote of associated Member Company. This procedure will prevent large companies with lots of Contributors from dominating the election results. 
```

Comment:
How will we determine which company a person is part of?

Reply:
We would need to have the Contributors self-identify. Is there already a list of Contributors to ONNX? I don't think I've seen one yet.